runtime/ops/os/mod.rs

```diff
@@ -186,6 +186,8 @@ fn op_get_exit_code(state: &mut OpState) -> i32 {
 
 #[op2(fast)]
 fn op_exit(state: &mut OpState) {
+  crate::ops::otel::otel_drop_state(state);
```

Comment:
I assume this is gonna be blocking/sending some requests. Might be worth adding a comment here. What happens if the program exits "cleanly"? Is this a best effort solution or do we want to guarantee that all data is flushed/sent to the collector?

Reply:
honestly I'm not sure what's up with this function, i would expect everything in OpState to be dropped correctly anyway so I'm not sure why it's needed, i was planning to investigate it later.

Reply:
Deno.exit() exits the process without running any destructor.